ramls/examples/check-in-by-barcode-response.json

```diff
@@ -11,7 +11,9 @@
         "name": "Checked Out"
       },
       "location": {
-        "name": "Main Library"
+        "name": "Main Library",
+        "code": "KU/CC/DI/M",
+        "library": "Datalogisk Institut"
```

Comment:
The name of the property in the schema is `libraryName`, it appears the example does not match that

Reply:
Changed